recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,13 @@
+###########################################################
+# Drop this file when the upstream issue is fixed.        #
+#                                                         #
+# xref: https://github.com/bacpop/pp-sketchlib/issues/92  #
+###########################################################
+c_compiler_version:            # [osx]
+  - 14                         # [osx]
+cxx_compiler_version:          # [osx]
+  - 14                         # [osx]
+libcxx:                        # [osx]
+  - 14                         # [osx]
+llvm_openmp:                   # [osx]
+  - 14                         # [osx]
```

Comment:
Filed issue ( https://github.com/conda-forge/pp-sketchlib-feedstock/issues/42 ) to track reverting this once the upstream issue ( https://github.com/bacpop/pp-sketchlib/issues/92 ) is resolved